### PR TITLE
SISRP-49141 - Award Comparison Card

### PR DESCRIPTION
**Pull request by marcpechaitis**
https://jira-secure.berkeley.edu/browse/SISRP-49141

### CODE REVIEW

File: app/models/edo_oracle/financial_aid/queries.rb [hunk 1]
```diff
@@ -5,21 +5,39 @@ class Queries < Connection
       include ClassLogger
       include Concerns::QueryHelper
 
-      def self.get_housing(person_id, aid_year)
+      def self.get_housing(person_id, aid_year:, effective_date: Time.zone.today.in_time_zone.to_date)
```
Comment (redconfetti) [inline app/models/edo_oracle/financial_aid/queries.rb:8]: @marcpechaitis - Where is the `aid_year` value here?

Comment (redconfetti) [reply app/models/edo_oracle/financial_aid/queries.rb:8]: Oh I see. Ruby 2.1 introduced required keyword arguments, which are [defined with a trailing colon](https://thoughtbot.com/blog/ruby-2-keyword-arguments).

File: src/redux/reducers/AwardComparisonSnapshotReducer.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      // newAidYears[action.aidYear] = { ...aidYears[action.aidYear] };
+      // newAidYears[action.aidYear][action.effectiveDate] = {
+      //   loaded: false,
+      //   error: null,
+      // };
+      // return { ...state, aidYears: newAidYears };
+      return appendEffectiveDateToAidYear(state, action, { loaded: false });
+
+    case FETCH_AWARD_COMPARISON_SNAPSHOT_SUCCESS:
+      // const aidYears = { ...aidYears };
```
Comment (redconfetti) [inline src/redux/reducers/AwardComparisonSnapshotReducer.js:40]: @marcpechaitis  - Are these comments needed?

Comment (marcpechaitis) [reply src/redux/reducers/AwardComparisonSnapshotReducer.js:40]: nope, I will remove

**Comment by redconfetti** [comment]
@marcpechaitis - Failing ESlint

```shell
> calcentral@51.0.0 lint /home/travis/build/sis-berkeley-edu/calcentral
> eslint .
/home/travis/build/sis-berkeley-edu/calcentral/src/react/components/_finances/AwardComparison/splitSubvalues.js
  9:1  error  Unexpected console statement  no-console
✖ 1 problem (1 error, 0 warnings)
```

File: src/react/components/_finances/AwardComparison/splitSubvalues.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const values = [
+  'Resident',
+  'Graduate/Professional',
+  '17 Units',
+  '10 Units',
+  'Rent & Utilities - LAW Prof',
+];
+
+console.log(values);
```
Comment (redconfetti) [inline src/react/components/_finances/AwardComparison/splitSubvalues.js:9]: @marcpechaitis - This is causing ESlint error

**Pull request merged**